deploy/backplane/mtsre/10-mtsre-credentials-request-apis.ClusterRole.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-mtsre-credentials-request-apis-reader
+rules:
+  - apiGroups:
+    - "cloudcredential.openshift.io"
+    resources:
+      - "credentialsrequests"
```

Comment:
Is there a specific `credentialsrequest` MTSRE needs access to?

Reply:
Nope, there's no dedicated `credentialsrequest` resource we'd need access. We need to all-in-all access to any `credentialsrequest` resource which CRO might be reconciling depending on the addon in the situation. Though we just require read-level access as we only need visibility to `credentialsrequest` CR for debugging purposes, therefore, there's no reason to have any write-level access.